pkg/parser/epub/parser_navdoc.go

```diff
@@ -88,21 +88,21 @@ func parseLiElement(li *xmlquery.Node, filePath string) (link *manifest.Link) {
 		title = strings.TrimSpace(muchSpaceSuchWowMatcher.ReplaceAllString(first.InnerText(), " "))
 	}
 	rawHref := first.SelectAttr("href")
-	href := "#"
+	href := url.MustURLFromString("")
```

Comment:
I don't think an empty string is a valid URL, does it pass the validation? We used a `#` when we needed a "null" URL.

Reply:
@mickael-menu yes, it parses with Golang's URL parser:
![image](https://github.com/user-attachments/assets/a7dcae27-29bf-41d9-ab8c-383b4fc626b1)
Do you want me to change it back to an anchor link?

Reply:
You decide what you think is best. IMHO it would be more consistent as `#` is used elsewhere in the Go toolkit.